packages/snap/src/utils/prompt.ts

```diff
@@ -1,20 +1,89 @@
-import { addressPage, balancePage } from '../dialogs';
-import { RpcMethods } from '@hathor/hathor-rpc-handler';
+import { addressPage, balancePage, changeNetworkPage, createNanoPage, createTokenPage, oracleDataPage, sendTransactionPage, signWithAddressPage, utxosPage } from '../dialogs';
+import { setNetwork } from '../utils/network';
+import { NETWORK_MAP } from '../constants';
+import { RpcMethods, TriggerTypes } from '@hathor/hathor-rpc-handler';
 
-export const promptHandler = (origin) => async (promptRequest) => {
+export const promptHandler = (origin, wallet) => async (promptRequest) => {
   const data = promptRequest.data;
   const params = promptRequest.params;
-  let confirmed;
-  switch (promptRequest.method) {
-    case RpcMethods.GetAddress:
-      confirmed = await addressPage(data, params, origin);
-      break;
-    case RpcMethods.GetBalance:
-      confirmed = await balancePage(data, params, origin);
+  let approved;
+  switch (promptRequest.type) {
+    case TriggerTypes.SignMessageWithAddressConfirmationPrompt:
+      approved = await signWithAddressPage(data, params, origin);
+      return { data: approved };
+    case TriggerTypes.PinConfirmationPrompt:
+      return {
+        data: {
+          accepted: true,
+          pinCode: '123',
```

Comment:
question: Should we use a constant or config for this value, so that it is easier to identify in future refactorings?